build/stage-build.yml

```diff
@@ -5,6 +5,9 @@ parameters:
   iosCertificateFile: ''
   iosProvisioningProfileFile: ''
   iosVariableGroup: ''
+- name: preBuildSteps
```

Comment:
what are those 3 lines for?

Reply:
It's the same as the others, except a little more detailed. If you look at the steps file, you'll notice. We could probably harmonize the format used here, just for clarity. Basically, any other line could be written
- name: iosCertificateFile
  type: string
  default: ''